lang/python/python3-readline/patches/001-remove-whitespace.patch

```diff
@@ -0,0 +1,11 @@
+--- a/setup.py
```

Comment:
Was this submitted to upstream? Why this is necessary at all to have it included in this repository? Does it throw any compilation error?

Reply:
No it doesn't. Tests won't accept this package without that patch because white-space in the end of line wasn't acceptable.

Reply:
That seems to me like an issue within tests because I can not imagine that contributor tries to add a package without any patches, and it blames him that there are whitespaces which he has not done.

Reply:
No, whitespace was in the original source.

Reply:
Isn't it what I said?